src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -159,55 +159,34 @@ const JobHistory = _.flow(
           },
           columns: [
             {
-              size: { basis: 600, grow: 0 },
-              headerRenderer: () => h(HeaderCell, ['Job']),
+              size: { basis: 500, grow: 0 },
+              headerRenderer: () => h(HeaderCell, ['Job (click for details)']),
               cellRenderer: ({ rowIndex }) => {
                 const {
-                  methodConfigurationNamespace, methodConfigurationName, submitter, submissionId, workflowStatuses,
-                  status, submissionEntity
+                  methodConfigurationNamespace, methodConfigurationName, submitter, submissionId
                 } = submissions[rowIndex]
                 return h(Fragment, [
                   div([
                     div([
                       methodConfigurationNamespace !== namespace && span({ style: styles.deemphasized }, [
                         `${methodConfigurationNamespace}/`
                       ]),
-                      methodConfigurationName
+                      link({
+                        target: '_blank',
+                        //replace link with actual job manager link
+                        href: linkToJobManager ? `${getConfig().jobManagerUrlRoot}/${namespace}/${name}/${submissionId}` :
```

Comment:
`${getConfig().jobManagerUrlRoot}?q=submission-id%3D${submissionId}` should work. (`%3D` is a URL-encoded `=` so that `q` has a value of `submission-id=blah`)